client/src/components/App/Content/InvestigationForm/TabManagement/PersonalInfoTab/PersonalInfoTab.tsx

```diff
@@ -1,20 +1,22 @@
-import React from 'react';
```

Comment:
I think the axios supposed  to be inside the investigation form component because this is the general component that contains all the tabs